Jenkinsfile.github

```diff
@@ -12,37 +12,38 @@ pipeline {
 
     environment {
         NAME = "cray-sts"
-        IMAGE_NAME = "cray-sts"
         DESCRIPTION = "Implements an STS service."
         IS_STABLE = getBuildIsStable()
-        IMAGE_VERSION = getDockerBuildVersion(isStable: env.IS_STABLE)
-        DOCKER_ARGS = getDockerBuildArgs(name: env.IMAGE_NAME, description: env.DESCRIPTION)
-        CHART_NAME = "cray-sts"
-        CHART_VERSION = getChartVersion(name: env.CHART_NAME, isStable: env.IS_STABLE)
+        VERSION = getDockerBuildVersion(isStable: env.IS_STABLE)
+        DOCKER_ARGS = getDockerBuildArgs(name: env.NAME, description: env.DESCRIPTION)
+        CHART_VERSION = getChartVersion(version: env.VERSION)
     }
 
     stages {
         stage("Build") {
             parallel {
-                stage("Image") {
+                stage('Image') {
                     steps {
                         sh "make image"
                     }
                 }
 
-                stage("Chart") {
+                stage('Chart') {
+                    environment {
+                        IMAGE = "artifactory.algol60.net/csm-docker/${env.IS_STABLE ? "stable" : "unstable"}/${env.NAME}"
+                    }
                     steps {
                         sh "make chart"
                     }
                 }
             }
         }
 
-        stage("Publish") {
+        stage('Publish ') {
```

Comment:
Is this extra space an issue?  'Publish '

Reply:
Shouldn't cause a problem, but fixed nonetheless.